test/framework/src/main/java/org/elasticsearch/test/rest/yaml/ESClientYamlSuiteTestCase.java

```diff
@@ -84,6 +85,7 @@
 
     private static final String DEFAULT_TESTS_PATH = "/rest-api-spec/test";
     private static final String DEFAULT_SPEC_PATH = "/rest-api-spec/api";
+    private static final String SETUP_YAML = "setup.yaml";
```

Comment:
It's leftover from my first implementation. For the record, my first implementation was to add a setup.yaml file that contain a top-level setup section that would be applied before every test. The problem with this is that there are *many* different rest test suites (in docs, in modules, in plugins, in smoke tests, etc.) and every single one of these would then need a duplicate copy of a setup.yaml that creates a template that sets the number of shards to one. After realizing this, I opted for the change here and this constant is just leftover.